README.md

```diff
@@ -13,7 +13,7 @@ Problem Bank
 
    1. Taking in input in multiple languages
       1. C++
-      2. Python
+      2. [Python](https://github.com/Samwest5/problem-bank/blob/master/pythoninput.md)
```

Comment:
We should use
```suggestion
      2. [Python](https://github.com/progteam/problem-bank/blob/master/pythoninput.md)
```
since this is the correct URL for people to see.

Reply:
Another thing to consider is to rename the file to `input_python.md` so we can add more languages and they will be sorted next to each other.